app/models/like.rb

```diff
@@ -0,0 +1,7 @@
+class Like < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+  def update_likes_counter
+    Post.update(LikesCounter: 1)
+  end
```

Comment:
### :red_circle: **[MANDATORY CHANGE]**

- [ ] It would be nice if you make your update_likes_counter as you can see below because the method you make is updating all posts' likes counter but it should only update the likes counter of that respective post for which any user has created a like. So it would be great if you make your method as below :+1:

```ruby
  def update_likes_counter
    post.increment!(:LikesCounter)
  end
```

### :yellow_circle: **[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - after_save to call the #update_likes_counter method every time a new like has been saved to the database, then make the update_likes_counter method private method :+1: You can see below what I am talking :+1:

```ruby
after_save :update_likes_counter

  private

  def update_likes_counter
    post.increment!(:LikesCounter)
  end
```